src/parsers/manifestjson.js

```diff
@@ -138,7 +142,7 @@ export default class ManifestJSONParser extends JSONParser {
 
       this.io = io;
       this.isAlreadySigned = isAlreadySigned;
-      this.isEnterpriseAddon = this.schemaValidatorOptions?.enterprise ?? false;
+      this.isEnterpriseAddon = isEnterprise;
```

Comment:
I noticed we didn't have to use the `schemaValidatorOptions` to pass this prop here.